src/test/java/io/druid/data/input/impl/FileIteratingFirehoseTest.java

```diff
@@ -0,0 +1,63 @@
+package io.druid.data.input.impl;
```

Comment:
Whoops, don't have that set up for druid-api. It seems a lot of people don't too… many other files are missing headers.